Backend/mentii/user_ctrl.py

```diff
@@ -10,6 +10,65 @@
 import hashlib
 from flask import g
 
+def sendForgotPasswordEmail(httpOrigin, jsonData, mailer, dbInstance):
+  email = jsonData.get('email', None)
+  resetPasswordId = str(uuid.uuid4())
+  addResetPasswordIdToUser(email, resetPasswordId, dbInstance)
```

Comment:
I want this function to fail completely silently as to not hint if a user exists or not. This has been reflected by always returning an empty response in app.py.